config/desktop/trixie/environments/cinnamon/armbian/create_desktop_package.sh

```
@@ -0,0 +1,164 @@
+# install default lightdm greeter settings
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)